### ISSUE TITLE
Config object injection fails in abstract class

### ISSUE THREAD

**Issue by smartwjw** [ctripcorp/apollo#1038]
In the following code snippet, config object named `config`  is null:

```Java
@Component
class AbstractT {
    @ApolloConfig
    private Config config;
}

@Service
class T extends AbstractT {
    @ApolloConfig
    private Config config1;
}
```

![image](https://user-images.githubusercontent.com/1625837/38450945-2d407108-3a59-11e8-952c-4c1c143bbc4b.png)

**Comment by nobodyiam** [comment]
Thanks! This issue has been fixed in v0.10.0, verified with unit tests in #1039.

**Comment by nobodyiam** [comment]
This case is closed for now. If there is any other problem, you could provide more information to this issue, thanks!
